### ISSUE TITLE
suggestion to use myanimelist for related animes and recommendations

### ISSUE THREAD

**Issue by ty-coon** [deceptions/no#6]
Anilist related anime and recommendations sucks
It would be better to use Mal api for it.

**Comment by deceptions** [comment]
I will check it out after I am done with Android TV and link priority.

**Comment by Bungeexgum** [comment]
I agree it's the same thing I don't like about shiro is they don't show recommendations/similar anime in the bottom. It's easy to know that anime has second seasons yo don't have to search it.
